web/src/main/resources/applicationContext-web.xml

```diff
@@ -25,10 +25,10 @@
       <bean name="/applicationSwaggerConfig" class="org.mskcc.cbio.portal.swagger.ApiServiceSwaggerConfig">
       </bean>
 
-    <context:component-scan base-package="org.mskcc.cbio.portal.web"/>
-    <context:component-scan base-package="org.mskcc.cbio.portal.documentation"/>
-    <context:component-scan base-package="org.cbioportal.web"/>
-    <context:component-scan base-package="org.cbioportal.weblegacy"/>
+    <context:component-scan base-package="org.mskcc.cbio.portal.web" />
```

Comment:
I guess these additional spaces are harmless .. but why not be consistent? the following lines below with "<mvc:resources ... " do not have an extra space before the closing of the tag.